main.js

```diff
@@ -552,6 +552,19 @@ app.on('ready', function() {
         require('open')(debugPath);
       });
     }},
+    {label: 'Send Debug Package', type: 'normal', click: function() {
+      var body = 'OpenBazaar Debug Report\n\n';
+      body += 'OS: ' + os.platform() + ' ' + os.release() + '\n';
+      body += 'Architecture: ' + os.arch() + '\n';
+      body += 'CPUs: ' + JSON.stringify(os.cpus(), null, 2) + '\n';
+      body += 'Free Memory: ' + os.freemem() + '\n';
+      body += 'Total Memory: ' + os.totalmem() + '\n\n';
+      body += 'Debug Log:\n';
+      body += serverOut;
+
+      require('open')('mailto:project@openbazaar.org?subject=OpenBazaar Debug Report&body=' + body);
```

Comment:
I use project@openbazaar.org as the default email since we control that, but users will probably send it wherever they will to get help.